Lib/test/test_exceptions.py

```diff
@@ -1765,272 +1885,6 @@ def test_name_error_has_name(self):
         except NameError as exc:
             self.assertEqual("bluch", exc.name)
 
-    def test_name_error_suggestions(self):
```

Comment:
Those tests were moved to `test_traceback`, but when I tried to update `test_traceback` from 3.13.5 nothing ran. Everything was skipped because we don't have `_testcapi`.

How should we proceed?

Reply:
Usually `_testcapi` is used for `@cpython_only` tests.
is PurePythonSuggestionFormattingTests also being skipped?

Reply:
>  is PurePythonSuggestionFormattingTests also being skipped?
Yes:/

Reply:
Could you also update test_traceback here? I will check if it is a bug of test suite or intended

Reply:
Now I see that it requires `has_debug_ranges`. which requires the `_testinternalcapi` :/

https://github.com/RustPython/RustPython/blob/a54873d302e2abb5d23d975006e9338d61bbc616/Lib/test/support/__init__.py#L505-L515